packages/manager/modules/hub/src/components/carousel/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ovh_cloud_hub_attention": "Attention: "
```

Comment:
are you sure we have to add this text before each notification? ("attention" means warning)

Reply:
The notifications listed are either `'warning'` or `'error'` which require attention of the customer as per specs